build.gradle

```diff
@@ -34,3 +34,57 @@ allprojects {
         ]
     }
 }
+
+// 0. clean up old files and register that task
+tasks.register('cleanUpChromeFolders', Delete) {
+    delete file('./chrome-binary'),
+            file('./chrome-driver')
+}
+tasks.named("clean") {
+    dependsOn('cleanUpChromeFolders')
+}
+
+// 1. create folders
+tasks.register('prepareDirs') {
+    mustRunAfter clean
+    doFirst {
+        mkdir "./chrome-binary/"
+        mkdir "./chrome-driver/"
+    }
+}
+
+// 2. download driver and binary
+tasks.register('downloadChromeDriver', Exec) {
+    dependsOn('prepareDirs')
+    commandLine 'curl', '-L', '-o', './chrome-driver/chromedriver_win32.zip',
+            'https://chromedriver.storage.googleapis.com/114.0.5735.90/chromedriver_win32.zip'
+}
+tasks.register('downloadChromeBinary', Exec) {
+    dependsOn('prepareDirs')
+    commandLine 'curl', '-L', '-o', './chrome-binary/chrome-win.zip',
```

Comment:
1. We _have to_ use _Chromium_ and not Chrome because group policies may prevent users from installing/using older Chrome versions.
Chromium has no such limitations.
2. Refer to https://chromium.cypress.io/ for Chromium versions
3. Re-upload chrome binary to realworld demo repository to avoid headache in case website from point above is unavailable